modules/cluster-logging-cloudwatch.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-openshift-cluster-monitoring.adoc
+// * logging/cluster-logging.adoc
+// * serverless/monitor/cluster-logging-serverless.adoc
+
+// This module uses conditionalized paragraphs so that the module
+// can be re-used in associated products.
+
+:_content-type: CONCEPT
+[id="cluster-logging-cloudwatch_{context}"]
+= CloudWatch recommendation for {product-title}
+
+Red Hat recommends that you use the AWS CloudWatch solution for your logging needs. 
+
+== Logging requirements
+
+Hosting your own logging stack requires a great deal of compute resources and storage, which may be dependent on your cloud service quota. The compute resource requirements can start at 48 GB and increase, while the storage requirement can sometimes be as large as 1,600 GB or more. The logging stack runs on your worker nodes, which reduce your available workload resource. With these considerations, hosting your own logging stack will increase your cluster operating costs.
```

Comment:
Should change "may" to "might"? And should the logging stack sentence say "which reduces"? And remove "will" so it should say "hosting your own logging stack increases your cluster operating costs" or "does increase", whichever sounds better.